static/src/javascripts/projects/common/modules/commercial/epic/iframe-epic-utils.js

```diff
@@ -42,7 +48,8 @@ const setIframeHeight = (epic: IframeEpicComponent, height: number) => {
 };
 
 const insertEpicIframe = (
-    epic: IframeEpicComponent
+    epic: IframeEpicComponent,
+    abTestVariant?: ABTestVariant
 ): Promise<IframeEpicComponent> =>
     new Promise((resolve, reject) => {
```

Comment:
Kinda outside the scope of this PR.. however it is probably worth checking the origin of the message? 🙏 

```
window.addEventListener('message', (event: MessageEvent) => {
    if (event.origin !== 'https://support.theguardian.com') {
        console.log('Hey I just met you, and this is crazy, unknown sender, so block me maybe?');
         return;
     }
```

I know that we are checking `if (!message || message.channel !== OPTIMIZE_EPIC_CHANNEL) {` but the origin check gives us some extra safety 😁